### PR TITLE
links ksrtc json with ksrtc page

### PR DESCRIPTION
**Pull request by naseemshah**
links ksrtc json with ksrtc page

image

### CODE REVIEW

File: src/components/Ksrtc/KsrtcCard.js [hunk 1]
```diff
@@ -3,46 +3,88 @@ import clockicon from "./assets/img/alarm-clock.svg";
...
-            <div class="flex text-center py-2 mt-0.5 space-x-2">
-                <div class="flex-1 border-2 border-black bg-black text-white font-normal text-sm text-center rounded-sm h-6.5 w-6.5">
+            <div className="flex text-center py-2 mt-0.5 space-x-2">
+                <div
+                    className={`flex-1 border-2 border-black ${
+                        props.availableDays === "allDay" ||
+                        props.availableDays.toLowerCase().includes("mon")
+                            ? "bg-black text-white"
+                            : "text-black"
+                    } font-normal text-sm rounded-sm h-6.5 w-6.5`}>
```
Comment (riginoommen) [inline src/components/Ksrtc/KsrtcCard.js:33]: If we can optimize this repeated code that can help increasing in performance

File: src/components/Ksrtc/KsrtcContainer.js [hunk 2]
```diff
@@ -1,19 +1,22 @@
...
-                        to={value.to}
-                        time={value.time}
-                        amorpm={value.amorpm}
-                        type={value.type}
+                        key={ksrtc.ksrtcId}
+                        from={ksrtc.from}
+                        to={ksrtc.to}
+                        time={new Date(ksrtc.kstrcTime).toLocaleString(
+                            "en-US",
+                            { hour: "numeric", minute: "numeric", hour12: true }
```
Comment (riginoommen) [inline src/components/Ksrtc/KsrtcContainer.js:16]: ```hour12``` is a not a right key

**Review by riginoommen — CHANGES_REQUESTED** [review]

File: src/components/Ksrtc/KsrtcContainer.js (hunk 2 above)
Comment (naseemshah) [reply src/components/Ksrtc/KsrtcContainer.js:16]: so remove the `hour12` ?

File: src/components/Ksrtc/KsrtcCard.js (hunk 1 above)
Comment (naseemshah) [reply src/components/Ksrtc/KsrtcCard.js:33]: oh lemme try to implement a function :)

File: src/components/Ksrtc/KsrtcContainer.js (hunk 2 above)
Comment (riginoommen) [reply src/components/Ksrtc/KsrtcContainer.js:16]: Can we rename it with ```twelveHour```

**Review by lespaceman — APPROVED** [review]

**Review by riginoommen — APPROVED** [review]

**Pull request merged**